versioned_docs/version-v3/guides/assets.md

```diff
@@ -209,7 +211,8 @@ declare function getAssetPath(path: string): string;
 ```
 
 The code sample below demonstrates the return value of `getAssetPath` for different `path` arguments, when an asset base path of `/build/` has been set.
-```ts
+import { getAssetPath } from '@stencil/core';
+
```

Comment:
I think this breaks the code styling - we lose the beginning of the code block here (it looks like this happens a few other places in this PR as well) - My guess is this is what's causing the MD link checks to fail as well. Since the code blocks are now 'offset', in that the ends of code blocks are now the starts of code blocks, pre-existing headers are assumed to be a part of a code block

Reply:
uff .. I think this is a copy&paste issue from my side.